pkg/providers/vsphere/template.go

```diff
@@ -23,7 +23,6 @@ func NewVsphereTemplateBuilder(
 ) *VsphereTemplateBuilder {
 	return &VsphereTemplateBuilder{
 		now:            now,
-		fromController: fromController,
```

Comment:
sorry, to be annoying
can we remove this from the template builder constructor?

Reply:
Also, I believe there is a test specifically for the `fromController: true` case
Can we remove that (and the expected file for it)?